### PR TITLE
Update focus-appearance-minimum.html

### LINKED ISSUE #1341: 2.4.11 Focus Appearance (Minimum) (AA) - clarification needed
Regarding the new sc 2.4.11 Focus Appearance (Minimum) (AA), I'd like clarification around the change of contrast requirement: "Change of contrast: The color change for the focus indication area has a contrast ratio of at least 3:1 with the colors of the unfocused state." 

If a focus indicator is added that 1.) does not rely on color and 2.) meets minimum adjacent contrast requirements, is it really necessary for the indicator to also have a 3:1 contrast with the unfocused state? 

If so, this contradicts past precedent of allowing for indicators besides color. It will also severely restrict the colors designers are able to use for default and focus states.

If not, could some clarifying language be added to this sc? 

Also, browser defaults have historically be treated as the exception, as they are not dictated by content creators. Will that continue to be the case regarding focus indicators? If so, language around that exception should be added.

### PR DESCRIPTION
**Pull request by mbgower**
Rewording of intro and all bullets except the last one, in an attempt to address https://github.com/w3c/wcag/issues/1341

Closes #1341, #1372

### CODE REVIEW

**Comment by awkawk** [comment]
I disagree that we should be including the formula for perimeter. This is well-understood outside of WCAG.

File: guidelines/sc/22/focus-appearance-minimum.html [hunk 1]
```diff
@@ -5,11 +5,11 @@ <h4>Focus Appearance (Minimum)</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>For the keyboard focus indicator of each User Interface Component, all of the following are true:</p>
+    <p>When user interface components receive keyboard focus, all of the following are true:</p>
```
Comment (awkawk) [inline guidelines/sc/22/focus-appearance-minimum.html:8]: Will need to be User Interface Component

Comment (alastc) [reply guidelines/sc/22/focus-appearance-minimum.html:8]: Someone tackled this in #1469, most are lower case so that PR was to make all lowercase.

**Comment by alastc** [comment]
Noting for the record that we discussed the perimeter aspect during the call, and there is an advantage to defining it normative, see #1324

**Pull request merged**